pkg/kubectl/clusterrolebinding_test.go

```diff
@@ -0,0 +1,215 @@
+				"serviceaccount": []string{"ns1:name1"},
+			},
+			expected: &rbac.ClusterRoleBinding{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "foo",
+				},
+				RoleRef: rbac.RoleRef{
+					APIGroup: rbac.GroupName,
+					Kind:     "ClusterRole",
+					Name:     "admin",
+				},
+				Subjects: []rbac.Subject{
+					{
+						APIGroup: rbac.GroupName,
+						Kind:     rbac.UserKind,
+						Name:     "user",
+					},
+					{
+						APIGroup: rbac.GroupName,
+						Kind:     rbac.GroupKind,
+						Name:     "group",
+					},
+					{
+						Kind:      rbac.ServiceAccountKind,
+						APIGroup:  "",
+						Namespace: "ns1",
+						Name:      "name1",
+					},
+				},
+			},
+			expectErr: false,
+		},
+		{
+			params: map[string]interface{}{
+				"name":           "foo",
+				"clusterrole":    "admin",
+				"user":           []string{"user1", "user2"},
+				"group":          []string{"group1", "group2"},
+				"serviceaccount": []string{"ns1:name1", "ns2:name2"},
+			},
+			expected: &rbac.ClusterRoleBinding{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "foo",
+				},
+				RoleRef: rbac.RoleRef{
+					APIGroup: rbac.GroupName,
+					Kind:     "ClusterRole",
+					Name:     "admin",
+				},
+				Subjects: []rbac.Subject{
+					{
+						APIGroup: rbac.GroupName,
+						Kind:     rbac.UserKind,
+						Name:     "user1",
+					},
+					{
+						APIGroup: rbac.GroupName,
+						Kind:     rbac.UserKind,
+						Name:     "user2",
+					},
+					{
+						APIGroup: rbac.GroupName,
+						Kind:     rbac.GroupKind,
+						Name:     "group1",
+					},
+					{
+						APIGroup: rbac.GroupName,
+						Kind:     rbac.GroupKind,
+						Name:     "group2",
+					},
+					{
+						Kind:      rbac.ServiceAccountKind,
+						APIGroup:  "",
+						Namespace: "ns1",
+						Name:      "name1",
+					},
+					{
+						Kind:      rbac.ServiceAccountKind,
+						APIGroup:  "",
+						Namespace: "ns2",
+						Name:      "name2",
+					},
+				},
+			},
+			expectErr: false,
+		},
+		{
+			params: map[string]interface{}{
+				"name":        "foo",
+				"clusterrole": "admin",
+			},
+			expected: &rbac.ClusterRoleBinding{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "foo",
+				},
+				RoleRef: rbac.RoleRef{
+					APIGroup: rbac.GroupName,
+					Kind:     "ClusterRole",
+					Name:     "admin",
+				},
+			},
+			expectErr: false,
+		},
+		{
+			params: map[string]interface{}{
+				"name":           "role",
+				"clusterrole":    "admin",
+				"user":           []string{"user"},
+				"group":          []string{"group"},
+				"serviceaccount": []string{"ns1-name1"},
+			},
+			expectErr: true,
+			reason:    "invalid serviceaccount, expected format: <namespace:name>",
+		},
+		{
+			params: map[string]interface{}{
+				"name":           "",
+				"clusterrole":    "admin",
+				"user":           []string{"user"},
+				"group":          []string{"group"},
+				"serviceaccount": []string{"ns1:name1"},
+			},
+			expectErr: true,
+			reason:    "name must be specified",
```

Comment:
I suggest to add a test for code here:
https://github.com/kubernetes/kubernetes/blob/877ee91930ccf980c7a396f956173bc55cad27c8/pkg/kubectl/clusterrolebinding.go#L154

When clusterrole == "", error happens.

Reply:
Done. Thanks!